src/doing/exceptions.py

```diff
@@ -34,4 +34,11 @@ def devops_error_tips(error: str) -> None:
     Azure devops can have quite technical errors where the solution is not immediately obvious.
     """
     if "The conditional access policy defined by your Azure Active Directory administrator has failed" in str(error):
-        console.print("[doing]: You might not be logged into azure. Try visiting dev.azure.com and signing in first.")
+        console.print("doing-cli: You might not be logged into azure. Try visiting dev.azure.com and signing in first.")
+
+    if "ERROR: TF401320: Rule Error for field Story Points." in str(error):
+        msg = "doing-cli: This error might be due to a policy enforcing work-items to have story points assigned.\n"
+        msg += "See the how to on bulk editing work items "
```

Comment:
I would rephrase to something like
`See an example how to bulk edit work items with no story points assigned yet in https://ing-bank.github.io/doing-cli/howto/workflow_bulk_edit_items`

Reply:
done